taipy/core/scenario/scenario.py

```diff
@@ -585,15 +585,15 @@ def submit(
     def export(
         self,
         folder_path: Union[str, pathlib.Path],
-        override: Optional[bool] = False,
-        include_data: Optional[bool] = False,
+        override: bool = False,
+        include_data: bool = False,
     ):
         """Export all related entities of this scenario to a folder.
 
         Parameters:
             folder_path (Union[str, pathlib.Path]): The folder path to export the scenario to.
-            override (Optional[bool]): If True, the existing folder will be overridden. Default is False.
-            include_data (Optional[bool]): If True, the file-based data nodes are exported as well.
+            override (bool): If True, the existing folder will be overridden. Default is False.
+            include_data (bool): If True, the file-based data nodes are exported as well.
                 This includes Pickle, CSV, Excel, Parquet, and JSON data nodes.
                 If the scenario has a data node that is not file-based, a warning will be logged, and the data node
                 will not be exported. The default value is False.
```

Comment:
```suggestion
                will not be exported. The default value is False.
        
        Raises:
            ExportFolderAlreadyExists^: If the `folder_path` already exists.
```

Reply:
What do you think about
```suggestion
                will not be exported. The default value is False.

        Raises:
            ExportFolderAlreadyExist^: If the `folder_path` already exists and the override parameter is False.
```

Reply:
Much better. Thx!